core/src/test/resources/current_user_metadata_access_control_tests/wiremock_service1_user5.json

```diff
@@ -11,6 +11,6 @@
   },
   "response": {
     "status": 200,
-    "body": "{ \"sth\": \"unknown\", \"groups\": [ { \"name\": \"service1_group1\" }, { \"name\": \"service1_group2\" } ] }"
+    "body": "{ \"sth\": \"unknown\", \"groups\": [ { \"id\": \"service1_group1\", \"name\": \"Group 1\"}, { \"id\": \"service1_group2\", \"name\": \"Group 2\" } ] }"
```

Comment:
It'd be better to leave the example as it was before (accurate to the name of the group ID). Let's create a new mock with the example above